cmd/owl2proto/owl2proto.go

```diff
@@ -189,51 +190,70 @@ func createProtoFile(preparedOntology ontology.OntologyPrepared, header string)
 	output += "\n\n" + header
 
 	// Create proto messages with comments
-	for _, v := range preparedOntology.Resources {
+	// Sort map keys
+	resources := util.SortMapKeys(preparedOntology.Resources)
+
+	for _, v := range resources {
```

Comment:
Maybe use "key" or "type" here instead of `v`, which is a bit confusing, since it's a key and not a value. Well its a value that is a key 🤯

Reply:
Better?